tests/e2e/analytics.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -euo pipefail
+
+TEST_NAME="analytics"
+
+cleanup() {
+  exit_code=$?
+  if [ "$exit_code" -ne 0 ]; then
+    echo "ERROR: '$TEST_NAME' tests failed during execution"
+    afterAll || echo "ERROR: Cleanup failed"
+  fi
+
+  exit "$exit_code"
+}
+trap cleanup EXIT
+
+beforeAll() {
+  echo "INFO: Executing '$TEST_NAME' tests"
+}
+
+beforeEach() {
+  rm -rf ./temp-config
+}
+
+afterAll() {
+  echo "INFO: Completed '$TEST_NAME' tests"
+  beforeEach
+}
+
+error() {
+  message=$1
+  echo "$message"
+  exit 1
+}
+
+beforeAll
+
+beforeEach
+
+# analytics defaults to enabled
+status="$("$DOPPLER_BINARY" analytics status --configuration=./temp-config --json)"
+[[ "$status" == '{"enabled":true}' ]] || error "ERROR: analytics not enabled"
+
+beforeEach
+
+# analytics defaults to enabled after clearing config
+"$DOPPLER_BINARY" configure reset --configuration=./temp-config --yes
```

Comment:
Nit: I think it would be better to test the `doppler configure reset` command *without* first deleting the existing temp-config file in `beforeEach`

Reply:
Good call, I'll add an additional test to verify `reset` works as expected when there's an existing config with analytics disabled.